From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Mon, 19 Jan 2026 10:11:27 -0600
Subject: [PATCH] No deb rpm

---
 app/build/Gruntfile.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/app/build/Gruntfile.js b/app/build/Gruntfile.js
index 44f935386..8af49df6e 100644
--- a/app/build/Gruntfile.js
+++ b/app/build/Gruntfile.js
@@ -58,6 +58,6 @@ module.exports = grunt => {
   } else if (grunt.option('platform') === 'darwin') {
     grunt.registerTask('build-client', ['package', 'create-mac-zip']);
   } else if (grunt.option('platform') === 'linux') {
-    grunt.registerTask('build-client', ['package', 'create-deb-installer', 'create-rpm-installer']);
+    grunt.registerTask('build-client', ['package']);
   }
 };
